tests/tests_app/components/database/test_client_server.py

```diff
@@ -141,22 +144,57 @@ def run(self):
                 self._client.insert(TestConfig(name="echo", secrets=[Secret(name="example", value="secret")]))
                 self._exit()
             else:
-                assert os.path.exists(id)
+                assert os.path.exists(self._db_filename)
                 assert len(self._client.select_all()) == 1
                 self._exit()
 
-    app = LightningApp(Flow())
-    MultiProcessRuntime(app).dispatch()
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
Another easy way to get a test-specific tmpdir is to use the predefined fixture like so:

```py
test_work_database_restart(tmpdir):
    ...
```

Reply:
oh I see, nice to know :-)